config/last_updated.json

```diff
@@ -31,8 +31,8 @@
   },
   "/static/js/exporting.js": {
     "date_published": "2021-05-19T00:00:00.000Z",
-    "date_modified": "2023-06-23T00:00:00.000Z",
-    "hash": "f36e2313c705b44e0779f7c34d9e34e0"
+    "date_modified": "2023-08-16T00:00:00.000Z",
+    "hash": "a2a9eaccff8895f1127a71d1821a18e4"
```

Comment:
FYI you don't need to update any of these (they are done with a GitHub Action just prior to release), and doing so is likely to lead to merge conflicts if other PRs are open at the same time and they also do this.